.github/workflows/deploy-to-eks.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
       
       - name: Test bundle deployment
         run: |
-          tox -vve test_bundle_deployment-${{ matrix.bundle_version }} -- --model kubeflow --keep-models -vv -s
+          tox -vve test_bundle_deployment-${{ matrix.bundle_version }} -- --model kubeflow --keep-models -vv -s --track ${{ matrix.bundle_version }}
```

Comment:
Also, this is not passing any `risk` argument.

Reply:
I think that if we want to go with this approach, then we would need to parse the `matrix.bundle_version` and split it into track and risk and always pass both, instead of relying to default values.